.gitignore

```diff
@@ -3,3 +3,4 @@ _site
 .sass-cache
 .jekyll-metadata
 vendor/
+.vscode
```

Comment:
This does not belong here

Reply:
Fair enough. This was to prevent extra files from being committed, but can remove that.